File: c/PROBLEMS/1150.c
#include <stdio.h>

int main(){
    int x, z, i;
    scanf("%d %d", &x, &z);

    while(z<=x){
        scanf("%d", &z);
    }
    int cont = 1, aux=x;
    while (aux<z){
        aux +=x;
        x+=1;
        cont++;
    }

    printf("%d\n", cont);

}

File: js/PROBLEMS/1150.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8')
var lines = input.split('\n')

lines = lines.map(x => parseInt(x))
x = lines[0]

for (i of lines){
    if (i > x){
        z=i
    }
}

aux = x
cont = 1

while (aux<z){
    aux +=x
    x+=1
    cont++
}
console.log(cont+'')